templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -5,13 +5,33 @@ param tags object = {}
 param containerAppsEnvironmentName string
 param containerName string = 'main'
 param containerRegistryName string
+
+@description('Minimum number of replicas to run')
+param containerMinReplicas int = 1
+@description('Maximum number of replicas to run')
+param containerMaxReplicas int = 1
+
 param env array = []
 param external bool = true
 param imageName string
 param keyVaultName string = ''
+@description('Param to denote whether to use a managed identity or not. If not, the identityPrincipalId output will be empty. DEPRECATED: Use managedIdentityEnabled instead.')
```

Comment:
Perhaps a question for @jongio - should we just remove this deprecated property as part of this PR? If not, what's the plan to remove it long term?

Reply:
I agree it is a better name.  If we take this, then we'll want to update all modules to use the same name.  This includes all bicep and terraform in azure-dev.  That can come as a diff PR if too large here, but don't just change it here.

Reply:
...and don't have both...

Reply:
No, let's not keep it.  It's a breaking change that we'll need to think about and flesh out.